src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -334,6 +404,8 @@ interface IGeneSelectionProps {
     ) => void;
     defaultNumberOfGenes: number;
     maxNumberOfGenes?: number;
+    comparisonStore?: ComparisonStore;
+    isForPathwayMapper?: boolean;
```

Comment:
Ideally we shouldn't need a flag like this one, this is too specific. We should rethink how to implement this functionality.